src/@chakra-ui/gatsby-plugin/components/Tag/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  createMultiStyleConfigHelpers,
+  theme,
+  defineStyle,
+} from "@chakra-ui/react"
+import { tagAnatomy } from "@chakra-ui/anatomy"
+import { $badgeColor, STATUS_COLORS } from "./utils"
+import { defineMergeStyles } from "../components.utils"
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(tagAnatomy.keys)
+
+const { Tag: tagTheme } = theme.components
+
+const baseStyleContainer = defineMergeStyles(tagTheme.baseStyle?.container, {
+  border: "1px",
+  borderColor: "transparent",
+  gap: 1,
+  borderRadius: "full",
+  px: 2,
+  minH: 8,
+  fontWeight: 300,
+  "&:any-link": {
+    textDecor: "none",
+    _focusWithin: {
+      outline: "4px solid",
+      outlineColor: "transparent",
+      outlineOffset: 0,
+    },
+  },
+})
+
+const baseStyleLabel = defineStyle({
+  ...tagTheme.baseStyle?.label,
+  fontSize: "xs",
+  textTransform: "uppercase",
+  textAlign: "center",
```

Comment:
To better align it vertically.

```suggestion
  textAlign: "center",
  lineHeight: 1,
```

Reply:
Right!